public/js/app.js

```diff
@@ -0,0 +1,315 @@
+            });
+            ScrollReveal().reveal('.punchline', {
+                delay: 1000
+            });
+
+            sr.reveal('.sc-item', {
+                rotate: {
+                    x: 100,
+                    y: 100,
+                    z: 0
+                },
+                duration: 1000
+            });
+
+            /**************** Menu Hamburger *****************/
+
+            var content = document.querySelector('#hamburger-content');
+            var sidebarBody = document.querySelector('#hamburger-sidebar-body');
+
+            sidebarBody.innerHTML = content.innerHTML;
+
+
+            var button = document.querySelector('#hamburger-button');
+            var overlay = document.querySelector('#hamburger-overlay');
+            var activatedClass = 'hamburger-activated';
+
+            button.addEventListener('click', function (e) {
+                e.preventDefault();
+
+                this.parentNode.classList.add(activatedClass);
+            });
+
+            button.addEventListener('keydown', function (e) {
+                if (this.parentNode.classList.contains(activatedClass)) {
+                    if (e.repeat === false && e.which === 27)
+                        this.parentNode.classList.remove(activatedClass);
+                }
+            });
+
+            overlay.addEventListener('click', function (e) {
+                e.preventDefault();
+
+                this.parentNode.classList.remove(activatedClass);
+            });
+
+            /********** to TOP bouton **********/
+
+            window.onscroll = function () {
+                scrollFunction()
+            };
+
+            function scrollFunction() {
+                if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+                    document.getElementById("toTop").style.display = "block";
+                } else {
+                    document.getElementById("toTop").style.display = "none";
+                }
+            }
+
+            /*************************HEADER**********************/
+
+            $('#header').hide();
+            var hauteur = 580;
+            $(function () {
+                $(window).scroll(function () {
+                    if ($(this).scrollTop() > hauteur) {
+                        $('#header').css("display", "flex")
+                         .fadeIn();
+                    } else {
+                        $('#header').hide();
+                    }
+                });
+            });
+
+            /********** Scroll smooth sections ************/
+
+            $(document).on('click', 'a[href^="#"]:not(li.nav-item a)', function (e) {
+                var id = $(this).attr('href');
+
+                var $id = $(id);
+                if ($id.length === 0) {
+                    return;
+                }
+
+                e.preventDefault();
+
+                var pos = $id.offset().top;
+
+                $('body, html').animate({
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/vanmarcke/ocr_projet5_blog/pullRequest?prid=8116981)
```suggestion
                $("body, html").animate({
```